doc/source/user_guide/boolean.rst

```diff
@@ -30,13 +31,6 @@ will raise a ``ValueError``.
    mask = pd.array([True, False, pd.NA], dtype="boolean")
    s[mask]
 
-The missing values will need to be explicitly filled with True or False prior
-to using the array as a mask.
```

Comment:
Can you keep this example but reword it as something like "if you want different behaviour, you can fill manually with fillna(True)" ?